xonsh/environ.py

```diff
@@ -926,6 +928,15 @@ def _current_job():
     return ''
 
 
+def env_name(pre_chars='(', post_chars=') '):
+    """Extract the current environment name from $VIRTUAL_ENV."""
+    env_path = builtins.__xonsh_env__.get('VIRTUAL_ENV', '')
```

Comment:
Fixed in https://github.com/moigagoo/xonsh/commit/926b74ba3f011ef6a264b91d9f49d3802f0d3add